x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccounts.java

```diff
@@ -22,6 +22,47 @@
 
     static final String NAMESPACE = "elastic";
 
+    private static final ServiceAccount ENTERPRISE_SEARCH_ACCOUNT = new ElasticServiceAccount(
+        "enterprise-search-server",
+        new RoleDescriptor(
+            NAMESPACE + "/enterprise-search-server",
+            new String[] { "manage", "manage_ilm", "manage_index_templates", "manage_security", "monitor" },
+            new RoleDescriptor.IndicesPrivileges[] {
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices(
+                        ".ent-search-*",
+                        ".monitoring-ent-search-*",
+                        "metricbeat-ent-search-*",
+                        "enterprise-search-*",
+                        "logs-app_search.analytics-default",
+                        "logs-enterprise_search.api-default",
+                        "logs-app_search.search_relevance_suggestions-default",
+                        "logs-crawler-default",
+                        "logs-workplace_search.analytics-default",
+                        "logs-workplace_search.content_events-default"
+                    )
+                    .privileges(
+                        "create",
+                        "create_index",
+                        "delete",
+                        "delete_index",
+                        "index",
+                        "manage",
+                        "manage_ilm",
+                        "monitor",
+                        "read",
+                        "view_index_metadata",
+                        "write"
+                    )
```

Comment:
The `manage` privilege covers `create_index`, `delete_index`, `manage_ilm`, `monitor` and `view_index_metadata`. The `write` privilege covers `create`, `delete` and `index`. So this list can be simplied down to `["read", "write", "manage"]`. 

Above privileges combined are very close to the `all` privilege. The only actions that are not granted are proxy CCS related ones, e.g. `internal:transport/proxy/indices:data/read/*`. Is CCS something enterprise search needs?

Reply:
> Is CCS something enterprise search needs?

I don't think we need it at this stage, so we can just change this to `["read", "write", "manage"]` as you suggested.